tests/fsharp/typecheck/sigs/neg104.vsbsl

```diff
@@ -19,9 +19,9 @@ neg104.fs(35,19,35,20): parse error FS0010: Unexpected symbol '}' in definition.
 
 neg104.fs(36,1,36,1): parse error FS0010: Incomplete structured construct at or before this point in implementation file
 
-neg104.fs(8,9,8,30): typecheck error FS0750: This construct may only be used within computation expressions
+neg104.fs(8,9,8,15): typecheck error FS0750: This construct may only be used within computation expressions
 
-neg104.fs(10,9,10,30): typecheck error FS0750: This construct may only be used within computation expressions
+neg104.fs(10,9,10,15): typecheck error FS0750: This construct may only be used within computation expressions
```

Comment:
Looking at these changes, do I get it right that `match!` range is also fixed now? :)

If so, it would be great to add a few tests for this as well, because it seems to me that we don't have anything about `match!` in the component tests.

(You can say no)

Reply:
I have a dedicated PR for `match!` in progress so if that is Ok I will add these there ?

Reply:
Nono, that's okay then, but it would be nice (and I am just curious) if you put a comment on what particular change in this PR affected this match! range.